xwiki-commons-core/xwiki-commons-velocity/src/main/java/org/xwiki/velocity/VelocityEngine.java

```diff
@@ -109,7 +102,10 @@ void evaluate(Context context, Writer out, String namespace, VelocityTemplate te
      *             sense anymore
      */
     @Deprecated
-    void clearMacroNamespace(String namespace);
+    default void clearMacroNamespace(String namespace)
+    {
+        
+    }
```

Comment:
Any reason for not moving those to legacy?

Reply:
Mostly lazyness to add aspectJ setup xwiki-commons-legacy-velocity and all the plumbing than goes with it.

Reply:
ok